build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'io.nextflow.nextflow-plugin' version '0.0.1-alpha2'
+}
+
+dependencies {
+    implementation 'org.json:json:20240303'
+    implementation 'dev.harrel:json-schema:1.5.0'
+    implementation 'com.sanctionco.jmail:jmail:1.6.3' // Needed for e-mail format validation
+}
+
+version = '2.4.0dev'
+
+nextflowPlugin {
+    nextflowVersion = '24.10.0'
```

Comment:
is this version required? We are using 24.04.2 in the nf-core pipelines template, this would force a bump of the minimal version 👀